src/separations.h

```diff
@@ -0,0 +1,131 @@
+#ifndef CYCAMORE_SRC_SEPARATIONS_H_
+#define CYCAMORE_SRC_SEPARATIONS_H_
+
+#include "cyclus.h"
+
+namespace cycamore {
+
+cyclus::Material::Ptr SepMaterial(std::map<int, double> effs, cyclus::Material::Ptr mat);
+
+class Separations : public cyclus::Facility {
+ public:
+  Separations(cyclus::Context* ctx);
+  virtual ~Separations(){};
+
+  virtual void Tick();
+  virtual void Tock();
+  virtual void EnterNotify();
+
+  virtual void AcceptMatlTrades(const std::vector<std::pair<
+      cyclus::Trade<cyclus::Material>, cyclus::Material::Ptr> >& responses);
+
+  virtual std::set<cyclus::RequestPortfolio<cyclus::Material>::Ptr>
+  GetMatlRequests();
+
+  virtual std::set<cyclus::BidPortfolio<cyclus::Material>::Ptr> GetMatlBids(
+      cyclus::CommodMap<cyclus::Material>::type& commod_requests);
+
+  virtual void GetMatlTrades(
+      const std::vector<cyclus::Trade<cyclus::Material> >& trades,
+      std::vector<std::pair<cyclus::Trade<cyclus::Material>,
+                            cyclus::Material::Ptr> >& responses);
+
+  #pragma cyclus clone
```

Comment:
I don't have a strong preference, but I do think it is sort of nice to have all pragmas all together (i.e. in the header file) rather than split across multiple.  I also don't feel strongly about needing to be consistent.  If you insist, however, I am happy to oblige.